docs/azure-saas-docs/content/en/quick-start.md

```diff
@@ -0,0 +1,76 @@
+
+
+## 2.b. Deploying to Azure - Entire Solution
+
+Deploying to Azure is easy thanks to our pre-configured ARM (Azure Resource Manager) templates.
+
+This button will take you to the Azure portal, passing it the template. You'll be asked a few questions, and then the solution will be up and running in just a few minutes. You will need your B2C configuration values and secrets from step 1.
+
+**Deployment Coming Soon!**
+<!-- [![Deploy to Azure](https://www.azuresaas.net/assets/images/deploy-to-azure.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2Fazuredeploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2FcreateUiDefinition.json) -->
+
+## 2.c. (Advanced) Deploying to Azure - Single Module
+
+If you'd like to use just one (or more) module from the project, we've provided bicep templates to do that as well. In each project directory, you'll find a folder named `{ModuleName}.Deployment` that contains all the bicep code you'll need to deploy just that Module. Please be advised that there are certain dependencies that each module requires in order for it to deploy properly. You may find that you need to edit the bicep templates to match your use case. You will find instructions and a list of dependencies for each module within the [module's readme](#more-info).
+
+## 3. (Optional) Configure Email Provider
+
+If you choose to deploy the entire solution to Azure, one of the things that gets deployed is the SaaS.Notifications module **need page and link**, an Azure Logic App. This logic app has a simple HTTP trigger that the SaaS.SignupAdministration site is configured to make a POST request to in order to send emails (ie, at the end of the tenant creation process). By default, there is no email provider configured and the emails do not actually get sent. If you'd like to enable email sending, you will need to manually configure your email provider connector of choice inside the Logic App. See the instructions [here](components/saas-notifications.md) to get started.
+
+## Integrating your application
+
+Now that you've seen how to build and run the code locally, and you have a way to deploy your code to Azure in a repeatable and code-first way, you can integrate your own code into the solution.
+
+We've included a basic application within the `Saas.Application.Web` project that demonstrates a SaaS solution called "BadgeMeUp". BadgeMeUp is simply a badge sharing site that *Contoso* (representing your company) can sell to end customers.
+
+> SaaS solutions come in many shapes as sizes. We picked "BadgeMeUp", because it's a fairly simple scenario to understand. [You can read more about this particular SaaS scenario here](../resources/contoso-badgemeup/).
+
+### How does this work?
```

Comment:
Since this gives further info about Bicep and isn't specific to integration, this section will probably be better suited if moved below 2.b. or 2.c.

Reply:
Alternatively, this could be changed to a Header2 equal with the sections above and below.